internal/admin/repository/main.tf

```diff
@@ -165,3 +151,33 @@ resource "github_issue_label" "ready_to_submit" {
   color       = "0e8a16"
   description = "Pull request has been approved and should be merged once tests pass"
 }
+
+resource "github_issue_label" "P0" {
+  repository  = "${github_repository.repo.name}"
+  name        = "P0"
+  color       = "990000"
+}
+
+resource "github_issue_label" "P1" {
+  repository  = "${github_repository.repo.name}"
+  name        = "P1"
+  color       = "ff6666"
+}
+
+resource "github_issue_label" "P2" {
```

Comment:
As mentioned offline, I'm nervous about including P2-P4. Can we omit these?

Reply:
Done.